Entities/ACraft.h

```diff
@@ -707,6 +707,7 @@ enum
                                  MOID rootMOID = g_NoMOID,
                                  bool makeNewMOID = true) { Actor::UpdateChildMOIDs(MOIDIndex, m_RootMOID, makeNewMOID); }
 
+    void SetAttachableVelocitiesForGibbing(Attachable* pAttachable, Vector impactImpulse, float internalBlast);
```

Comment:
Needs method description

Reply:
Method description is in cpp file since otherwise vs doesn't seem to populate its tooltips